docs/advanced.rst

```diff
@@ -493,6 +493,91 @@ For example, you might implement JSON PATCH according to `RFC 6902 <https://tool
         """
         # ...
 
+Multi-Field Detection
+---------------------
+
+If a ``List`` field is used to parse data from a location like query parameters --
+where one or multiple values can be passed for a single parameter name -- then
+webargs will automatically treat that field as a list and parse multiple values
+if present.
+
+To implement this behavior, webargs will examine schemas for ``marshmallow.fields.List``
+fields. ``List`` fields get unpacked to list values when data is loaded, and
+other fields do not. This also applies fields which inherit from ``List``.
+
+.. note::
+
+    In webargs v8, ``Tuple`` will be treated this way as well, in addition to ``List``.
+
+What if you have a list which should be treated as a "multi-field" but which
+does not inherit from ``List``? webargs offers two solutions.
+You can add the custom attribute `is_multiple=True` to your field or you
+can add your class to your parser's list of `KNOWN_MULTI_FIELDS`.
+
+First, let's define a "multiplexing field" which takes a string or list of
+strings to serve as an example:
+
+.. code-block:: python
+
+    # a custom field class which can accept values like List(String()) or String()
+    str_instance = fields.String()
+
+
+    class CustomMultiplexingField(fields.Field):
```

Comment:
Why the instance? Couldn't this inherit from String using super to access String methods?

Reply:
Yes, it could/should!
I had just imitated the implementation of `List` (which has a nested field instance) without thinking too hard.

I'll simplify it.